src/plugins/irc/irc-bar-item.c

```diff
@@ -772,6 +831,12 @@ irc_bar_item_init ()
 
     weechat_hook_focus ("buffer_nicklist",
                         &irc_bar_item_focus_buffer_nicklist, NULL, NULL);
+    weechat_hook_focus ("buflist",
+                        &irc_bar_item_focus_buflist, NULL, NULL);
+    weechat_hook_focus ("buflist2",
+                        &irc_bar_item_focus_buflist, NULL, NULL);
+    weechat_hook_focus ("buflist3",
+                        &irc_bar_item_focus_buflist, NULL, NULL);
```

Comment:
This is slightly copy-paste-ish but that's the only way in irc plugin to take care of all the buflists without having a weird dependency on the buflist plugin's constant. This could be improved by allowing wildcards in `hook_focus` area names.